pac4j-core/src/main/java/org/pac4j/core/client/BaseClient.java

```diff
@@ -206,10 +211,18 @@ public void setProfileCreator(final ProfileCreator<C, U> profileCreator) {
         this.profileCreator = profileCreator;
     }
 
+    public Map<String, Object> getCustomProperties() {
+        return customProperties;
+    }
+
+    public void setCustomProperties(final Map<String, Object> customProperties) {
+        this.customProperties = customProperties;
```

Comment:
Can we just check that the `customProperties` is not `null`?

Reply:
Sure. Done.